base.js

```diff
@@ -1,25 +1,33 @@
 module.exports = {
-    parser: '@typescript-eslint/parser', // Specifies the ESLint parser
     env: {
-        node: true,
         es6: true,
+        node: true,
     },
     extends: [
         'eslint:recommended',
         'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
         'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
     ],
+    parser: '@typescript-eslint/parser', // Specifies the ESLint parser
     parserOptions: {
         ecmaVersion: 6, // Allows for the parsing of ES6 features
         sourceType: 'module', // Allows for the use of imports
     },
     rules: {
         // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+        '@typescript-eslint/camelcase': 'off',
         // Public is assumed by default for class members, and shouldn't be explicit
-        '@typescript-eslint/explicit-member-accessibility': 'no-public',
+        '@typescript-eslint/explicit-member-accessibility': 'off',
```

Comment:
The `no-public` value was throwing an error in the core app.  I did some quick research on the plugin and came across this https://github.com/typescript-eslint/typescript-eslint/issues/422

Reply:
@itslikegood is it possible to make this update in a separate PR? I'd like to run a release w/ this update as it's affecting other apps

Reply:
Sorry David! I didn't follow up on this PR because I figured Bryan wouldn't be back for some time.  We're gonna get this merged now, I hope this isn't still blocking.